keras/losses.py

```diff
@@ -1496,6 +1500,10 @@ def mean_absolute_percentage_error(y_true, y_pred):
   diff = tf.abs(
       (y_true - y_pred) / backend.maximum(tf.abs(y_true),
                                           backend.epsilon()))
+  if ignore_zeros is True:
```

Comment:
Just use "if ignore_zeros" (remove the is True check). The implicit check is preferred and more readable in this case.